src/boards/include/boards/pico.h

```diff
@@ -81,4 +81,19 @@
 #define PICO_RP2040_B0_SUPPORTED 1
 #endif
 
+// GPIO pin get VBUS
+#ifndef PICO_VBUS_GPIO_PIN
+#define PICO_VBUS_GPIO_PIN 24
+#endif
+
+// GPIO pin used to monitor VSYS
+#ifndef PICO_VSYS_GPIO_PIN
+#define PICO_VSYS_GPIO_PIN 29
+#endif
+
+// ADC to use to monitor VSYS
+#ifndef PICO_VSYS_ADC_NUM
+#define PICO_VSYS_ADC_NUM 3
```

Comment:
I wonder if `PICO_VSYS_ADC_CHANNEL` might be less ambiguous, as a name ending in `_NUM` _might_ be misinterpreted to mean "count of"? (IYSWIM!)  :shrug:

Reply:
Looking at the api "input" might be better.